test/createAzureResource.test.ts

```diff
@@ -120,6 +122,37 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         const deletedApp: WebSiteManagementModels.Site | undefined = await webSiteClient.webApps.get(resourceName, resourceName);
         assert.ifError(deletedApp); // if app was deleted, get() returns null.  assert.ifError throws if the value passed is not null/undefined
     });
+
+    test('Create a new windows Web app and deploying zips to Web App', async () => {
+        const resourceGroupName: string = getRandomHexString();
+        const webAppName: string = getRandomHexString();
+        const AppServicePlan: string = getRandomHexString();
+        resourceGroupsToDelete.push(resourceGroupName);
+        const testInputs: string[] = [getTestRootZipFile(), '$(plus) Create new Web App...', webAppName, '$(plus) Create new resource group', resourceGroupName, 'Windows', '$(plus) Create new App Service plan', AppServicePlan, 'S1', 'West US', 'Deploy'];
```

Comment:
This is going to have to be refactored like this 
https://github.com/microsoft/vscode-azurefunctions/blob/master/test/nightly/createProjectAndDeploy.test.ts#L54-L55